packages/cms-drupal/src/Entity/PageListBuilder.php

```diff
@@ -43,10 +43,11 @@ public static function createInstance(ContainerInterface $container, EntityTypeI
     public function buildHeader(): array
     {
         return [
-            'id' => $this->stringTranslation->translate('Machine name'),
-            'title' => $this->stringTranslation->translate('Title'),
-            'path' => $this->stringTranslation->translate('Path'),
-        ] + parent::buildHeader();
+                'id' => $this->stringTranslation->translate('Machine name'),
+                'title' => $this->stringTranslation->translate('Title'),
+                'path' => $this->stringTranslation->translate('Path'),
+                'channels' => $this->stringTranslation->translate('Channels'),
```

Comment:
Just for the information.
`PageListBuilder` extends `EntityListBuilder` which extends `EntityHandlerBase`.
`EntityHandlerBase` use the `StringTranslationTrait` so you could just use `$this->t()` if you want.
"Thanks" to this trait you already have the `$stringTranslation` property (it just need to be set in the constructor).

Reply:
will do